Misc/NEWS.d/next/Library/2022-10-31-12-34-03.gh-issue-98897.rgNn4x.rst

```diff
@@ -0,0 +1 @@
+Fix memory leak in :func:`math.dist` when it raises exception. Patch by Kumar Aditya.
```

Comment:
```suggestion
Fix memory leak in :func:`math.dist` when both points don't have the same dimension. Patch by Kumar Aditya.
```

I don't think that it's useful to mention the fix on the unlikely MemoryError code path.